modules/argocd/Makefile

```diff
@@ -1,15 +1,31 @@
-RELEASE_TAG ?= 0.1.0
-.PHONY: updater/test check-prereq-variables
+RELEASE_TAG ?= 1.0.0
+dirs ?= rendered/
+envs ?=
+apps ?= ""
+
+dev_dir = rendered/environments/aws.dev
+qa_dir = rendered/environments/aws.qa
+logs_dir = rendered/environments/aws.logs
+prod_dir = rendered/environments/aws.prod
```

Comment:
It might be good to filter on `app` and `env` the same way `make k8s/tanka/generate` [does it](https://github.com/mintel/build-harness-extensions/blob/main/modules/k8s/tanka/generate.sh#L86) (using `tk env list --names`). In fact, we could probably pull that out into a "list with filters" common script.

Using that, you wouldn't need to filter by app in `argocd-image-updater-tester`; you could just pass in a list of directories:

```
@bin/argocd-image-updater-tester --dirs "$$(list_with_filters.sh --app '$(app)' --env '$(env)' | paste -d, -)"
```